cluster-autoscaler/core/scaledown/unneeded/nodes.go

```diff
@@ -30,11 +30,17 @@ import (
 	"k8s.io/autoscaler/cluster-autoscaler/simulator"
 	"k8s.io/autoscaler/cluster-autoscaler/utils"
 	kube_util "k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes"
+	"k8s.io/autoscaler/cluster-autoscaler/utils/taints"
 
 	apiv1 "k8s.io/api/core/v1"
 	klog "k8s.io/klog/v2"
 )
 
+const (
+	// NODE_COOLDOWN_SINCE_ANNOTATION is the annotation used to store the time when a node was found to be unneeded.
+	NODE_COOLDOWN_SINCE_ANNOTATION = "cluster-autoscaler.kubernetes.io/unneeded-since"
```

Comment:
I don't think we should have this as an annotation, but for future reference: Please use CamelCase, we don't use ALL_CAPS for constants generally.